src/migrations/20210909085651-add-protected-field-to-environments.js

```diff
@@ -0,0 +1,22 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+    ALTER TABLE environments ADD COLUMN protected BOOLEAN DEFAULT FALSE;
+    UPDATE environments SET protected = TRUE WHERE name = ':global:';
```

Comment:
```suggestion
    ALTER TABLE environments ADD COLUMN protected BOOLEAN DEFAULT false;
    UPDATE environments SET protected = true WHERE name = ':global:';
```
Preference thing, and a very tiny nit, but I prefer the booleans to be lower rather than uppercase, but it doesn't really matter, we're not looking at this very often :)